src/NovakSolutions/Infusionsoft/Model/Model.php

```diff
@@ -91,6 +91,8 @@ public function fromArray($data){
                     foreach($value as $object){
                         $this->{$key}[] = new $fieldDetails[1]($object);
                     }
+                } elseif (strpos($fieldDetails, "NovakSolutions\Infusionsoft\Model") === 0){
```

Comment:
This has the potential to not do what you expect if there is a new namespace like ModelSomething.  I would put the trailing slash on that if you can.